src/main/java/org/folio/qm/service/EventProcessingService.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.qm.service;
+
+import org.folio.rest.jaxrs.model.DataImportEventPayload;
+
+public interface EventProcessingService {
+
+  boolean processDICompleted(DataImportEventPayload data);
```

Comment:
It is not immediately clear what these methods return. And one service not use the result at all.
So I think we should stay with returning `void` in both of methods